src/main/java/com/github/onsdigital/babbage/search/external/SearchEndpoints.java

```diff
@@ -1,11 +1,11 @@
 package com.github.onsdigital.babbage.search.external;
 
-import com.github.onsdigital.babbage.search.external.requests.search.requests.ListType;
+import com.github.onsdigital.babbage.search.external.requests.search.ListType;
 
 public enum SearchEndpoints {
 
     SEARCH("/search/"),
-    SEARCH_ONS("/search/%s/"),
+    SEARCH_ONS("/search/"),
```

Comment:
Do we still need `SEARCH_ONS`, can we not just use `SEARCH` now that the proxying is gone?

Reply:
Removed SEARCH_ONS entry